src/e2o/defaultStartup.js

```diff
@@ -0,0 +1,83 @@
+// Electron does not yet support "import"
+const { app } = require('electron');
+const e2o = require("@chartiq/e2o")
+//const FinsembleElectronUpdater = require('./src/startup/finsembleElectronUpdater');
+const log = require('electron-log');
+const superagent = require('superagent');
+const fs = require('fs');
+const MANIFEST_URL = "http://localhost:3375/configs/openfin/manifest-local.json"
```

Comment:
Should you get this from the application's config in case it's different from our default?